src/base/preferences.h

```diff
@@ -115,16 +115,18 @@ class Preferences: public QObject
     void setHideZeroValues(bool b);
     int getHideZeroComboValues() const;
     void setHideZeroComboValues(int n);
+#ifndef Q_OS_MAC
```

Comment:
I would take out all entries in this `#ifndef` block and move it down to where you put
```c++
#ifndef Q_OS_MAC
     TrayIcon::Style trayIconStyle() const;
```